apps/openmw/mwmechanics/actors.cpp

```diff
@@ -420,6 +420,15 @@ namespace MWMechanics
 
             if (LOS)
```

Comment:
You should add "{" after this line or remove "}" from 431 line to compile OpenMW properly.